src/cloud.rs

```diff
@@ -0,0 +1,24 @@
+use crocksdb_ffi::{self};
```

Comment:
Gate the whole file with `feature = "cloud"`

Reply:
I thought it was already being conditionally compiled as per [lib.rs#L74](https://github.com/de-sh/rust-rocksdb/blob/32a65dbb479921a31015767d652ee47ede966930/src/lib.rs#L74)